cocos/platform/CCApplication.h

```diff
@@ -101,7 +101,7 @@ class CC_DLL Application
     // This class is useful for internal usage.
     static Application* getInstance() { return _instance; }
     
-    Application(const std::string& name, int width, int height);
+    Application(const std::string& name, int width, int height, bool fullscreen);
```

Comment:
It will break API. I think use default value is better.

Reply:
Yes, already added.